### DISCUSSION TITLE
Is there a risk? Should we take risks, how much risk can we bear, and is the possible income worth bearing these risks? How many people do we need to participate in the decision to take this risk? How to reduce the risk?

### DISCUSSION THREAD

**Discussion by lets-make-safe-ai** [lets-make-safe-ai/make-safe-ai#19] [General]
Is there a risk? Should we take risks, how much risk can we bear, and is the possible income worth bearing these risks? How many people do we need to participate in the decision to take this risk? How to reduce the risk?

**Comment by chirag127** [comment]
job loss is the biggest risk
